test/JDBC/expected/declare_atatglobalvars_upgrade-before-16_5-vu-verify.out

```diff
@@ -0,0 +1,16 @@
+exec p1_declare_atatglobalvars_upgrade
```

Comment:
Why does this not exist post upgrade? Ideally the prepare script would run on 16.4 and when we upgrade to 16.5/16.6 then the object should still exist

Reply:
The upgrade would fail as the object cannot be recreated anymore

Reply:
Object wont be recreated from 16.4 to 16.5/16.6, even if they did, the restore happens from psql endpoint and ANTLR does not come into picture. Dump/restore happens only for Major version upgrade.